cisagov/ScubaGoggles
# [#395] Update Calendar policy 4.1 to detect inherited settings

## 🗣 Description ##

Update Calendar policy 4.1 to detect inherited settings

### 💭 Motivation and context

Calendar policy should inherit the setting from the parent OU


## 🧪 Testing

1. Enable paid appointments in an OU other than the primary OU
2. Run ScubaGoggles, note that GWS.CALENDAR.4.1 is now failing (as expected)
3. Return to the admin center, click "INHERIT" for that setting for that OU.
4. Run ScubaGoggles again, note that the OU is listed as compliant (as expected).

## ✅ Pre-approval checklist ##




## ✅ Pre-merge Checklist


## ✅ Post-merge Checklist
